converter/mpi_cog_convert.sh

```diff
@@ -58,31 +60,39 @@ find "$SRCDIR" -name "*.nc" | \
             echo "$file" >> "$FILEL$j"
             i=$((i+1))
         fi
-        if [ $((i)) -gt $((NCPUS)) ]
+        if [ $((i)) -gt $((NCPUS*20)) ]
```

Comment:
This loop really needs some comments about what it's trying to achieve. I'm pretty sure it's splitting up the job so that there are 20 NetCDF files per CPU to process, and at the same time filtering out empty NetCDF files.

If we are actually running efficiently using MPI, we may not need to submit multiple PBS jobs.

If we *do* need to submit multiple jobs, and we want things to be equally sized, we should take into account the number of output GeoTIFFs, since this will vary *greatly* depending on the number of _times_ and the number of _measurements_ in the input files. Unstacked WOFs will have 1 output GeoTIFF per NetCDF files, stacked fractional cover will have ~ 100 output GeoTIFFs per NetCDF file...